core-bundle/src/Resources/contao/languages/en/tl_module.xlf

```diff
@@ -426,7 +426,7 @@
         <source>Confirmation page</source>
       </trans-unit>
       <trans-unit id="tl_module.reg_jumpTo.1">
-        <source>Please choose the page to which users will be redirected after the request has been completed.</source>
+        <source>Please choose the page to which visitors will be redirected after the request has been completed.</source>
```

Comment:
Why did you not use "members" here? The `reg_jumpTo` field defines the page on which **members** can confirm their registrations.